api/database_access/interfaces.py

```diff
@@ -77,56 +77,77 @@ class MongoInterface(DBInterface):
   survey_id = "atlas"
 
   @classmethod
-  def get_light_curve(cls, object_id):
-    light_curve = {
-      "detections": cls.get_detections(object_id),
-      "non_detections": cls._get_non_detections(object_id)
-    }
-
-    for det in light_curve["detections"]:
-      det["phase"] = 0  # (det.mjd % obj.period) / obj.period
-
-    return light_curve
+  def _get_object(cls, object_id):
+    astro_object = mongo_db.query().find_one(
+      model=mongo_models.Object,
+      filter_by={
+        "oid": object_id
+      }
+    )
+    if astro_object:
+      return astro_object
+    else:
+      raise ObjectNotFound(object_id=object_id, survey_id=cls.survey_id)
 
   @classmethod
-  def get_detections(cls, object_id):
-    """
-      There is no object without detections
-    """
+  def _get_detections(cls, object_id):
     detections = mongo_db.query().find_all(
       model=mongo_models.Detection,
       filter_by={
-        "oid": object_id
+        "aid": object_id
```

Comment:
In this query we also need to filter by `tid`, in case that there are detections coming from ZTF